jhucourse/woj/backend/game.py

```diff
@@ -0,0 +1,134 @@
+'import json'
+import numpy
+import player
+'import QuestionBank'
+
+class game:
```

Comment:
Nit: Classes are capitalized so call is Game

Reply:
Understood, I'll make the change.